app/assets/javascripts/discourse/app/lib/d-popover.js

```diff
@@ -24,18 +24,23 @@ export const hideOnEscapePlugin = {
   },
 };
 
+export function isPopoverShown(event) {
```

Comment:
Maybe this should be named `isPopoverVisible`? Given that it uses `isVisible` as opposed to `isShown`? (alternatively: should use `isShown`? as the functions below do?)

Reply:
@CvX ah, good catch! We should consistently use tippy's internal `isShown` state (instead of `isVisible`), and then the functionality and the function name are all in alignment.